multiqc/modules/interop/interop.py

```diff
@@ -195,137 +206,198 @@ def run_metrics_summary_table(self, data):
             'title': 'Surface',
             'description': ''
         }
         headers['Tiles'] = {
             'title': 'Tiles',
-            'description': 'The number of tiles per lane.'
+            'description': 'The number of tiles per lane.',
+            'hidden': True
         }
         headers['Density'] = {
-            'title': 'Density (K / mm^2)',
-            'description': 'The density of clusters (in thousands per mm2) detected by image analysis, +/- 1 standard deviation.'
+            'title': 'Density',
+            'description': 'The density of clusters (in thousands per mm2) detected by image analysis, +/- 1 standard deviation.',
+            'hidden': True
         }
         headers['Cluster PF'] = {
             'title': 'Cluster PF (%)',
-            'description': 'The percentage of clusters passing filtering, +/- 1 standard deviation.'
+            'description': 'The percentage of clusters passing filtering, +/- 1 standard deviation.',
+            'suffix': '%',
+        }
+        headers['Phased'] = {
+            'title': 'Phased (%)',
+            'description': 'The value used by RTA for the percentage of molecules in a cluster for which sequencing falls behind (phasing) or jumps ahead (prephasing) the current cycle within a read.',
+            'format': '{:.,2f}',
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'OrRd'
         }
-        headers['Phas/Prephas'] = {
-            'title': 'Phas/Prephas (%)',
-            'description': 'The value used by RTA for the percentage of molecules in a cluster for which sequencing falls behind (phasing) or jumps ahead (prephasing) the current cycle within a read.'
+        headers['Prephased'] = {
+            'title': 'Prephased (%)',
+            'description': 'The value used by RTA for the percentage of molecules in a cluster for which sequencing falls behind (phasing) or jumps ahead (prephasing) the current cycle within a read.',
+            'format': '{:.,2f}',
+            'min': 0,
+            'max': 100,
+            'suffix': '%',
+            'scale': 'OrRd'
         }
         headers['Reads'] = {
             'title': 'Reads',
-            'description': 'The number of clusters (in millions).'
+            'description': 'The number of clusters (in millions).',
+            'suffix': 'M',
         }
         headers['Reads PF'] = {
             'title': 'Reads PF',
-            'description': 'The number of clusters (in millions) passing filtering.'
-        }
-        headers['%>=Q30'] = {
-            'title': '%>=Q30',
-            'description': 'The percentage of bases with a quality score of 30 or higher, respectively.'
-        }
-        headers['Yield'] = {
-            'title': 'Yield',
-            'description': 'The number of bases sequenced which passed filter.'
+            'description': 'The number of clusters (in millions) passing filtering.',
+            'suffix': 'M',
```

Comment:
Please see the other comment below for how to handle table columns with read counts.